### ISSUE TITLE
mkapi does not detect classmethods on abstract base class

### ISSUE THREAD

**Issue by holmrenser** [daizutabi/mkapi#18]
mkapi does not seem to detect classmethods in my setup.

Simplified example:
```python
from abc import ABCMeta, abstractmethod
import json

class Base(object, metaclass=ABCMeta):
    @abstractmethod
    def __init__(self, some_arg):
        """[summary]

        Args:
            some_arg ([type]): [description]

        Raises:
            NotImplementedError: [description]
        """
        raise NotImplementedError()

    @classmethod
    def from_json(cls, json_string):
        """[summary]

        Args:
            json_string ([type]): [description]

        Raises:
            NotImplementedError: [description]
        """
        raise NotImplementedError()

    def to_json(self):
        """[summary]
        """
        return json.dumps(self.data)


class Child(Base):
    def __init__(self, some_arg):
        """[summary]

        Args:
            some_arg ([type]): [description]
        """
        self.some_arg = some_arg
        self.data = None

    @classmethod
    def from_json(cls, json_string):
        """[summary]

        Args:
            json_string ([type]): [description]
        """
        child = cls()
        child.data = json.loads(json_string)
        return child        
```

This should render docs for both classes where they both have `from_json` and `to_json`. Instead the classmethod `from_json` is missing from both.

EDIT: when removing the `metaclass=ABCMeta` from the baseclass the classmethods pop up. Is this described behaviour?

**Comment by daizutabi** [comment]
Thanks for reporting this issue.
This bug has been fixed in Version 1.0.9. 
See the current behavior [here](https://mkapi.daizutabi.net/appendix/method/#abstract-class).

**Comment by holmrenser** [comment]
Upgrading to 1.0.9 fixed this issue, thanks!
